doc/about.rst

```diff
@@ -0,0 +1,14 @@
+PennyLane-Qiskit Plugin
+#######################
```

Comment:
I noticed and deleted the `PennyLane-Qiskit` page. What can we do so that clicking on the `PennyLane-Qiskit` side bar heading actually loads the index page?

Reply:
I think it can be done, let me make a commit that adds it :)

Reply:
Done! I've made an edit to `doc/xanadu_theme/globaltoc.html`, adding in a new section above the existing toctree:

![image](https://user-images.githubusercontent.com/2959003/78517627-1cb0c700-77fd-11ea-800e-3defeba94a6d.png)

Unfortunately, looks like we can't 'insert' the homepage inside the existing toctree, as sphinx autogenerates the html of the entire toctree in one go.

Reply:
Genius :) You mind I change it to "About"?

Reply:
Go for it, that sounds better!